kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/resources/KubernetesResourcesApplier.java

```diff
@@ -0,0 +1,65 @@
+package org.arquillian.cube.kubernetes.impl.resources;
+
+import io.fabric8.kubernetes.api.model.v3_1.HasMetadata;
+import io.fabric8.kubernetes.clnt.v3_1.KubernetesClient;
+import org.arquillian.cube.kubernetes.annotations.KubernetesResource;
+import org.arquillian.cube.kubernetes.impl.adapter.KubernetesAdapter;
+import org.arquillian.cube.kubernetes.impl.utils.RunnerExpressionParser;
+import org.jboss.arquillian.core.api.annotation.Observes;
+import org.jboss.arquillian.test.spi.TestClass;
+import org.jboss.arquillian.test.spi.event.suite.AfterClass;
+import org.jboss.arquillian.test.spi.event.suite.BeforeClass;
+
+import java.io.BufferedInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class KubernetesResourcesApplier {
```

Comment:
Can we add support for method scope? So you can define a Kubernetes resource at the method level. In Istio we are not offering this because I am not sure that it has sense at all since it is a really high-level routing feature.